lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -144,6 +145,16 @@ function bucketLevelServerSideEncryption(bucketName, headers, log, cb) {
         });
 }
 
+function isLegacyAwsBehavior(locationConstraint, usEastBehavior) {
```

Comment:
please add jsdoc.  also, will you have to use a similar function elsewhere?  if so, extract into a utility file.